### PR TITLE
module-1

### PR DESCRIPTION
**Pull request by anton-ocheret**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/anton-ocheret/node-course/issues"
+  },
+  "homepage": "https://github.com/anton-ocheret/node-course#readme",
+  "devDependencies": {
+    "nodemon": "^2.0.7"
```
Comment (mshalabaiev-newscorp) [inline package.json:22]: nodemon is installed but not used

File: task1.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+rl.on('line', (input) => {
+  process.stdout.write(`${input.split('').reverse().join('')}\n`);
```
Comment (mshalabaiev-newscorp) [inline task1.js:9]: `console.log` will be enough and no need of `\n`

File: task2.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const { pipeline } = require('stream');
+const fs = require('fs');
+const csv = require('csvtojson');
```
Comment (mshalabaiev-newscorp) [inline task2.js:3]: csvtojson is not added as a dependency, task2 desn't work.

**Review by mshalabaiev-newscorp — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (anton-ocheret) [reply package.json:22]: I just don't see a purpose of using nodemon in these tasks, fixed

File: task1.js [hunk 4]
```diff
@@ -6,5 +6,5 @@ const rl = readline.createInterface({
 });
 
 rl.on('line', (input) => {
-  process.stdout.write(`${input.split('').reverse().join('')}\n`);
+  console.log(`${input.split('').reverse().join('')}`);
```
Comment (mshalabaiev-newscorp) [inline task1.js:9]: No need of backticks.

**Review by mshalabaiev-newscorp — CHANGES_REQUESTED** [review]

**Review by mshalabaiev-newscorp — APPROVED** [review]

**Pull request merged**